libs/designsystem/calendar/src/calendar.component.ts

```diff
@@ -548,4 +525,76 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
     const numberOfYears = endYear - startYear;
     return Array.from({ length: numberOfYears + 1 }, (_, i) => (startYear + i).toString());
   }
+
+  private setFocussedCell(newDate: Date) {
+    const newDay = this.getCell(newDate);
+    if (newDay) {
+      if (this.focussedDay) {
+        this.focussedDay.isFocussed = false;
+      }
+      newDay.isFocussed = true;
+      this.focussedDay = newDay;
+    }
```

Comment:
Consider guarding same was as `focusDate` method below:
```suggestion
    if (!newDay) return;
    if (this.focussedDay) {
        this.focussedDay.isFocussed = false;
     }
     newDay.isFocussed = true;
     this.focussedDay = newDay;
```